src/api/index.js

```diff
@@ -282,3 +282,7 @@ export const createImage = ({
 export const fetchEdgeImages = ({ limit = 100, offset = 0 } = {}) => {
   return instance.get(`${EDGE_API}/images?limit=${limit}&offset=${offset}`);
 };
+
+export const sortImageColunm = ({ sortColunm }) => {
```

Comment:
no good reason to create a new action - you have `fetchEdgeImages`. Use it for that.